### ISSUE TITLE
Roundstart threat report is broken, and meta-able.

### ISSUE THREAD

**Issue by lordpidey** [tgstation/tgstation#30338]
The roundstart centcom threat report is broken, and meta-able.

Right now, if the gamemode is changeling, traitor, bloodcult, revolution, or wizard, the roundstart report will have an extra horizontal line (Total of eight), and the current gamemode WILL be in the report.  So you can look on the report and ignore any entries other than the five I listed before, which will likely narrow the possible gamemodes down to one or two.

If the gamemode is NOT any of those five modes, it is entirely random what is on the report, making the report entirely useless.

Further, the report still has the deprecated gamemode "Malf" as a possibility on it, so that threat can be totally ignored.

Bottom line:  The report can either be used to narrow down the roundtype with frightening accuracy, or is completely useless.

**Comment by AnturK** [comment]
Let's we just remove this "feature" and put some random mode-unrelated stories in there.

**Comment by lordpidey** [comment]
I feel the ability to narrow down threats some will lead to extra paranoia.  Sometimes knowing what to look for increases paranoia, not decreasing it.
